Automattic/wxr-generator
# Remove extra space

The extra space will break when testing in PHP 8, removing it will fix the problem.

Test instructions:

- Setting up wxr generator with substack importer using PHP 8 and run an substack import to see if it works without a Fatel error.
